sender_test.go

```diff
@@ -186,17 +187,19 @@ func TestInt64Serialization(t *testing.T) {
 
 func TestLong256Column(t *testing.T) {
 	ctx := context.Background()
-
+	
+	val, _ := big.NewInt(0).SetString("ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff", 16)
 	testCases := []struct {
 		name     string
-		val      string
+		val     *big.Int
 		expected string
 	}{
-		{"random bits ", "0x0123a4i", "0x0123a4"},
-		{"8bit max", "0xffffffffi", "0xffffffff"},
-		{"16bit max", "0xffffffffffffffffi", "0xffffffffffffffff"},
-		{"32bit max", "0xffffffffffffffffffffffffffffffffi", "0xffffffffffffffffffffffffffffffff"},
-		{"64bit long", "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffi", "0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff"},
+		{"zero", big.NewInt(0), "0x0"},
+		{"one", big.NewInt(1), "0x1"},
+		{"32-bit max", big.NewInt(math.MaxInt32), "0x7fffffff"},
+		{"64-bitrandom",big.NewInt(7423093023234231), "0x1a5f4386c8d8b7"},
```

Comment:
Please reformat the code:
```suggestion
		{"64-bitrandom", big.NewInt(7423093023234231), "0x1a5f4386c8d8b7"},
```